src/sagemaker/jumpstart/factory/estimator.py

```diff
@@ -381,7 +385,7 @@ def _add_region_to_kwargs(kwargs: JumpStartKwargs) -> JumpStartKwargs:
 
 def _add_sagemaker_session_to_kwargs(kwargs: JumpStartKwargs) -> JumpStartKwargs:
     """Sets session in kwargs based on default or override, returns full kwargs."""
-    kwargs.sagemaker_session = kwargs.sagemaker_session or Session()
+    kwargs.sagemaker_session = kwargs.sagemaker_session or DEFAULT_JUMPSTART_SAGEMAKER_SESSION
```

Comment:
sanity-check: wouldn't this default all the sessions to `us-west-2` ?

Reply:
Only if they don't have any region set in the AWS config, which is usually not the case or only done in error.